lib/usd/utils/CMakeLists.txt

```diff
@@ -84,3 +92,75 @@ if(${CMAKE_SYSTEM_NAME} MATCHES "Windows")
             DESTINATION ${CMAKE_INSTALL_PREFIX}/lib OPTIONAL
     )
 endif()
+
+# -----------------------------------------------------------------------------
+# python
+# -----------------------------------------------------------------------------
+
+set(PY_INIT_FILES
+  ${CMAKE_CURRENT_SOURCE_DIR}/__init__.py
+)
+
+# copy init files to build location for module tests
+foreach(INPUT_FILE ${PY_INIT_FILES})
+  string(REPLACE ${CMAKE_CURRENT_SOURCE_DIR} ${CMAKE_CURRENT_BINARY_DIR}/${TARGET_NAME} OUTPUT_FILE ${INPUT_FILE})
+  execute_process(COMMAND ${CMAKE_COMMAND} -E copy ${INPUT_FILE} ${OUTPUT_FILE})
+endforeach()
```

Comment:
move all these after the call to `add_library`. 

Use `mayaUsd_copyFiles `. There are couple of issues with this kind of copy.

We want to copy resources before the target has been built, hence `PRE_BUILD` and `if and only something has changed`, hence `copy_if_different`.

`PRE_BUILD` COMMAND ${CMAKE_COMMAND} -E `copy_if_different`

https://github.com/Autodesk/maya-usd/blob/f34c6ed315ae509a06b3a054c2a38a1c65f870b9/cmake/utils.cmake#L222